ui/src/components/Splashscreen.tsx

```diff
@@ -66,9 +66,8 @@ export default function Splashscreen() {
       <div id="error" className={classes.error}>
         <div className={classes.errorFlex}>
           <j-text variant="heading-lg">Whoops, something broke! 😅</j-text>
-          <j-text variant="ingress">If you have also been asked to include a log file with your report,
-            click the button below to copy a log file to your desktop:</j-text>
-          <j-button variant="primary" onClick={copyFile}>{copied ? "Copied" : "Copy"}</j-button>
+          <j-text variant="ingress">To help us fix this, please click the button below to open your AD4M data folder. Please then send the ad4min.log file found there to us on Discord.</j-text>
```

Comment:
Just changed that file name to ad4m.log in the this PR: https://github.com/perspect3vism/ad4m/pull/147
```suggestion
          <j-text variant="ingress">To help us fix this, please click the button below to open your AD4M data folder. Please then send the ad4m.log file found there to us on Discord.</j-text>
```